plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -2309,6 +2309,25 @@ struct DbCoverage {
         return rval;
     }
 
+    CovRegionStatus[][FileId] getCoverageStatus() @trusted {
```

Comment:
I think you should change it to taking a FileId as argument. because it is only ever used to get all regions in a file. not all regions that exist.